README.md

```diff
@@ -36,7 +36,11 @@ Lita.configure do |config|
   config.robot.adapter = :slack
   config.robot.admins = ["U012A3BCD"]
   config.adapters.slack.token = "abcd-1234567890-hWYd21AmMH2UHAkx29vb5c1Y"
-  config.adapters.slack.parse = "full" # See https://api.slack.com/docs/formatting#parsing_modes
+  config.adapters.slack.postMessage = {
+    parse: "full" # See https://api.slack.com/docs/formatting#parsing_modes
+    # or link_names: 1
+    # or unfurl_links: true
```

Comment:
IMO the doc should propose what will be the next major version's default. And IMO the sanest defaults are: `{link_names: 1, unfurl_links: false}`